addons/OPT/GPS/fn_IsUnitLeader.sqf

```diff
@@ -28,12 +28,14 @@
 
 params ["_unit"];
 
-switch (typeOf _unit) do {
-	case "OPT_NATO_Gruppenfuehrer";
-	case "OPT_CSAT_Gruppenfuehrer";
-	case "OPT_NATO_Offizier";
-	case "OPT_CSAT_Offizier";
-	case "OPT_NATO_Pilot";
-	case "OPT_CSAT_Pilot": { true };
-	default { false };
-};
\ No newline at end of file
+(typeof _unit in [
+    "OPT_NATO_Gruppenfuehrer", 
+    "OPT_CSAT_Gruppenfuehrer",
+    "OPT_NATO_Offizier",
+    "OPT_CSAT_Offizier",
+    "OPT_NATO_Pilot",
+    "OPT_CSAT_Pilot"
+]
+|| roleDescription _unit in [
+    "GF-Crew@Lowe"
+] )
```

Comment:
role-descriptions ändern sich ja wahrscheinlich immer mal wieder (zB bei einem Kampagnen-Wechel). Darum könnte man hier vllt stattdessen mit einer Variable arbeiten, die dann für die entsprechenden Einheiten in der init gesetzt wird. Würde dann halt voraussetzen, dass die Missionsbauer das genauso brav machen, wie die Rollenbeschreibung.
Vorteil wäre, dass das Prozedere über alle Kampagnen hinweg das selbe bleibt und nichts mehr in dieser Funktion geändert werden muss (das hat auch den netten Nebeneffekt, dass die Mod auch für ältere Kampagnen verwendet werden kann, obwohl sich ggf. die Rollenbeschreibungen inzwischen geändert haben).

```
_unit getVariable [QGVAR(unitIsLeader), false]
```

Reply:
Das roleDescription behindert ist, weiß ich, ist aber unserem höchst merkwürdigem Kit/Slot-System geschuldet ist. init und Variable wäre sicher logischer, aber du hast es schon erkannt, Missionsbauer ;) .
Ich gehe davon aus, dass wir uns sehr bald von diesen Ausnahmen lösen werden, so dass tatsächlich nur noch die bestimmten unittypes leader sind, wie vorgesehen. Zumindest, bis wir irgendwann mal ein vernünftiges Kit-&Gruppensystem haben *träum* .